Exercise_D_E/assets/style/index.css

```diff
@@ -2,3 +2,88 @@
 This css sheet is linked to practice.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+* {
+    margin: 0;
+    padding: 0;
+    font-style: "meiryo";
+  }
+
+.canvas{
+    background: rgba(0,0,0,0.7) url(../images/flowershop.jpg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-blend-mode: darken;
+    display: flex;
+    opacity: 0.9;
+    width:100%; 
+    height: 600px;
+}
+
+.btntext{
+    position: absolute;
+    top:40%;
+    left: 35%;
+    font-size: 3em;
+    color: white;
+    text-shadow: black;
+}
+
+.btn {
+    display: block;
+    margin: 0 auto;
+    margin-top: 20px;
+    background-color: green;
+    color: white;
+    font-size: 1.5rem;
+    padding: 15px;
+    border-radius: 30px;
+    border: 1px solid black;
+  }
+
+.infocontainer1{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: wrap;
+  }
+
+.infocontainer2{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: wrap;
+  }
+
+.info1{
+    background-color: #c8e5aa;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width:50%;
+    padding: 50px 0px
+  }
+
+
+.info2{
+    background-color: #c8e5aa;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width:50%;
+    padding: 50px 0px
+}
+
+.text1{
+    font-size: 25px;
+    margin-left: 40px;
+    opacity: 0.8;
+}
+
+.text2{
+    font-size: 25px;
+    margin-left: 40px;
+    opacity: 0.8;
+}
+
+
+  
```

Comment:
You have recreated the website exactly like how it's supposed to be with fantastic use of flex and background blend mode. Excellent work!